travel-event-host-app/src/app/page.tsx

```diff
@@ -1,6 +1,12 @@
 import Categories from '@/components/categories/categories';
 import Image from 'next/image';
+import Footer from '@/components/footer/Footer';
 
 export default function Home() {
-  return <Categories />;
+  return (
+    <div>
+      <Categories />
+      <Footer />
```

Comment:
Great Job, but the Footer go in the layout, put it just below of the {children}

Reply:
Thanks for the review! Done